### ISSUE TITLE
Battery Status: Remaining Battery Time incorrectly Output

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#10146]
### Steps to reproduce:

1. On a device that has a battery, ensure that it is not connected to power and wait until the battery level is at around 30%

2. Activate the NVDA gesture assigned to reading the battery status

### Actual behavior:

NVDA will announce something along the lines of "battery percentage 1 hours 20 minutes remaining"

### Expected behavior:

To make this gromatically correct, this should instead read as "battery percentage 1 hour 20 minutes remaining" (the "s" after the word "hour" should be removed if NVDA detects that only a single hour is estimated)

**Comment by lukaszgo1** [comment]
Duplicate of #9003 though this issues suggest more improvements to this script. Given that someone besides me reported it I'll probably submit a PR for this.

**Comment by XLTechie** [comment]
In fact, it shouldn't say "percentage" at all, when what it is giving is a time 
estimate. A percentage would be correct if it was saying "Battery percentage 
30%".
Even in that context, the word "percentage" is redundant.
For either of these, I would expect:

Battery 30% remaining.

or:

Battery 1 hour 2 minutes remaining.

**Comment by lukaszgo1** [comment]
@XLTechie This script announces battery percentage, and then remaining time.

**Comment by leonardder** [comment]
Note that the script is using an outdated API to get this info. There is a new API which presents info that resembles the info from the system tray icon.

**Comment by josephsl** [comment]
Hi,

Six years later...

Is the new API the one used in more recent NVDA releases? If yes, then I think we should close this issue as resolved.

Thanks.

**Comment by SaschaCowley** [comment]
Looks like this was actually fixed in ad0fe837268c92b5df4368a82dc6ed551bf49629. Closing as complete.
